xboxone/xboxone.py

```diff
@@ -111,7 +114,7 @@ def volume_controls(self):
         if not volume_controls:
             return None
 
-        controls = volume_controls.get('avr') or volume_controls.get('tv')
+        controls = volume_controls.get('tv')
```

Comment:
Why remove AVR? This would mean volume controls won’t work if someone has an external sound bar.

Reply:
I did that because the volume controls weren't working for my tv, so I thought that might fix it but it hasn't.  I was planning on working on that next.  I put the original line back in.  Does the volume work with your setup? Do you have it working with an AVR or tv?

I just submitted an update to fix the status.  When the console is on it was showing up as unknown in the history.  I just removed "not" from the state property.